source/databricks/tests/integration/conftest.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture(scope="session")
+def spark() -> SparkSession:
+    return SparkSession.builder.config(
+        "spark.sql.streaming.schemaInference", True
+    ).getOrCreate()
```

Comment:
I find this much easier to read:
```suggestion
    return (SparkSession
               .builder
               .config("spark.sql.streaming.schemaInference", True)
               .getOrCreate())
```